package.json

```diff
@@ -28,7 +28,7 @@
   },
   "build": {
     "productName": "Mark Text",
-    "appId": "org.simulatedgreg.electron-marktext",
```

Comment:
A little confused what's the `appId` used for?

Reply:
> The application id. Used as CFBundleIdentifier for MacOS and as Application User Model ID.aspx) for Windows (NSIS target only, Squirrel.Windows not supported). It is strongly recommended that an explicit ID is set.

Basically it's a application identification id. Maybe user have to uninstall old Mark Text, when we change `appId`. I'll try that later.